### PR TITLE
Removing static variables from EnzoBlock

### PR DESCRIPTION
**Pull request by mabruzzo**
Removed remaining static variables from ``EnzoBlock``. These variables all either:
- held redundant information that was already stored elsewhere
- hold configuration parameters for the ppm solver.

I simply removed attributes in the former case and changed attributes in the latter case so that they are now stored as attributes of `EnzoMethodPpm`

### CODE REVIEW

File: src/Enzo/hydro-mhd/SetMinimumSupport.cpp [hunk 1]
```diff
@@ -12,10 +12,15 @@
 //----------------------------------------------------------------------
  
 int EnzoBlock::SetMinimumSupport(enzo_float &MinimumSupportEnergyCoefficient,
+                                 enzo_float minimum_pressure_support_parameter,
 				 bool comoving_coordinates)
 {
   const int in = cello::index_static();
```
Comment (gregbryan) [inline src/Enzo/hydro-mhd/SetMinimumSupport.cpp:18]: Is this line required at all anymore?

File: src/Enzo/hydro-mhd/EnzoMethodPpm.cpp [hunk 2]
```diff
@@ -265,7 +288,7 @@ double EnzoMethodPpm::timestep ( Block * block ) throw()
 			&enzo_block->CellWidth[0],
 			&enzo_block->CellWidth[1],
 			&enzo_block->CellWidth[2],
-			&gamma, &EnzoBlock::PressureFree[in], &cosmo_a,
+			&gamma, &pressure_free_int, &cosmo_a,
```
Comment (gregbryan) [inline src/Enzo/hydro-mhd/EnzoMethodPpm.cpp:291]: I think this change means you can remove the line setting `in`

**Review by gregbryan — APPROVED** [review]
This PR looks good to me.

File: src/Enzo/hydro-mhd/EnzoMethodPpm.cpp (hunk 2 above)
Comment (mabruzzo) [reply src/Enzo/hydro-mhd/EnzoMethodPpm.cpp:298]: good catch! I removed it!

File: src/Enzo/hydro-mhd/SetMinimumSupport.cpp (hunk 1 above)
Comment (mabruzzo) [reply src/Enzo/hydro-mhd/SetMinimumSupport.cpp:18]: This argument is used  is used inside of `SetMinimumSupport`. Specifically, the argument is multiplied all of the other variables that are used to compute `MinimumSupportEnergyCoefficient`

**Review by gregbryan — APPROVED** [review]
Approved!

**Pull request merged**